modules/mapper-extras/src/yamlRestTest/resources/rest-api-spec/test/match_only_text/10_basic.yml

```diff
@@ -351,3 +351,45 @@ tsdb:
         "@timestamp" : "2000-01-01T00:00:00.000Z"
         "dimension" : "a"
         foo: "Apache Lucene powers Elasticsearch"
+
+---
+synthetic_source with copy_to:
+  - requires:
+      cluster_features: ["mapper.source.synthetic_source_restrictions_removed"]
+      reason: requires copy_to support in synthetic source
+
+  - do:
+      indices.create:
+        index: synthetic_source_test
+        body:
+          mappings:
+            _source:
+              mode: synthetic
+            properties:
+              foo:
+                type: match_only_text
```

Comment:
if here we add `store: true`...are we going to store the value twice?

Reply:
Yes but it is not necessarily the same value (e.g. normalizers).